src/plugins/intel_cpu/src/infer_request.cpp

```diff
@@ -626,7 +643,23 @@ void InferRequest::initBlobs() {
         InferRequest::GetBlob(it.first);
     }
     for (const auto& it : modelOutputsMap) {
-        InferRequest::GetBlob(it.first);
+        auto outblob = InferRequest::GetBlob(it.first);
+
+        outputAllocators[it.first] = std::make_shared<OutputAllocator>(outblob);
```

Comment:
The dependency must be inverted. The blob should be initialized with the allocator, not vice versa.